tests/util/caches/test_descriptors.py

```diff
@@ -470,9 +470,9 @@ def func(self, key):
 
         a = A()
 
-        a.func.prefill(("foo",), ObservableDeferred(d))
```

Comment:
I don't think that prefilling with an ObservableDeferred is the way this API is meant to be used. At least, it hasn't been used that way for a long time.